spec/catalog.yaml

```diff
@@ -102,7 +102,40 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/namespaces/{ns}:
+    parameters:
+      - $ref: '#/components/parameters/namespace'
+
+    get:
+      tags: [ Namespace ]
+      summary: Load the metadata properties for a namespace
+      operationId: loadNamespaceMetadata
```

Comment:
nit: what about `GetNamespace`? To make the operation ID consistent with the response model name

Reply:
sounds reasonable..